02-peer-protocol.md

```diff
@@ -50,10 +67,451 @@ pubkey corresponding to the funding output nothing prevents duplicative channel
 ids.
 
 
+### `channel_id`, v2
+
+For channels established using the v2 protocol, the `channel_id` is the
+SHA256(lesser-revocation-basepoint || greater-revocation-basepoint),
+where the lesser and greater is based off the order of the basepoint. The basepoints are compact DER-encoded public keys.
+
+
+## Interactive Transaction Construction
+
+Interactive transaction construction consists of allowing two peers to share inputs
+and outputs for a transaction. This protocol is the foundation for dual-funded
+channels establishment, splicing, multi-purposes closes, as well as
+multi-party (3+ nodes) transactions.
+
+### Set-Up and Vocabulary
+
+There are two parties to an interactive construction: an `initiator` and a `contributor`.
+The `initiator` is the peer which initiates the protocol, e.g. for channel establishment
+the `initiator` would be the peer which sends `open_channel`; for a close,
+the `initiator` sends `shutdown`.
+
+The protocol that follows makes the following assumptions:
+
+- The `feerate` for the transaction has already been agreed upon.
+- The `nLocktime` for the transaction has already been communicated, or is
+  established via convention.
+- The transaction version is 2.
+- The each input's sequence number will be set to 0xFDFFFFFF (little endian), which signals RBF.
+- The initiator has furnished a PoDLE commitment, if required.
```

Comment:
I think that since this is the Set-Up and Vocabulary section it might be nice to spell out what PoDLE is and link to its definition since it is used and referenced pretty extensively before it is defined